src/app/admin/AdminPage.tsx

```diff
@@ -6,13 +6,22 @@ import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
 import { ExpandMore } from "@mui/icons-material";
 import TableSurface from "@/components/Tables/TableSurface";
 import CreateUserForm from "@/app/admin/createUser/CreateUserForm";
-import { faUsers, faUserPlus, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+import {
+    faUsers,
+    faUserPlus,
+    faCity,
+    faBuildingCircleArrowRight,
+    IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
 import styled from "styled-components";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { UserRow } from "./page";
+import { UserRow } from "@/app/admin/page";
+import CollectionCentresTable from "@/app/admin/collectionCentresTable/CollectionCentresTable";
+import { Schema } from "@/database_utils";
+import CreateCollectionCentreForm from "@/app/admin/createCollectionCentre/CreateCollectionCentreForm";
 
-const PanelIcon = styled(FontAwesomeIcon)`
-    margin-right: 0.5em;
+const IconWrapper = styled.div`
```

Comment:
just curious - did the previous implementation not work?

Reply:
Oh originally there was an icon that was particularly narrow and if I don't use this then the width will be different and the word will go too left
But now I realise the current one are all of same size so all good i removed it